processes/WIP-bug_bounty_proposal.md

```diff
@@ -0,0 +1,73 @@
+# Proposal to advance a Bug Bounty program for the Node.js® organization
+
+## Introduction
+
+Security is an essential part of the ongoing success of the Node.js community. The community is a vital part of one of the most aggressive markets available providing critical runtime infrastructure for a growing percentage of the internet and corporate computing requirements in 2018. A bug bounty program has become a standard mechanism to improve the security posture of projects, organizations and companies at every level.
+
+A bug bounty program is a mechanism to formally engage security researchers and developers in a way that provides benefits to the community along multiple vectors:
+
+1. Improved Security Depth of Coverage • Finding security defects is specialized work that benefits from deep expertise and fresh eyes
+1. Validate Nodejs as a secure platform • Expanding the community into the security world is a growth area and enables new audiences for nodejs to be proposed
```

Comment:
s/Nodejs/Node.js

Reply:
then later in this sentence s/nodejs/Node.js

(that is, make sure all of the references to Node.js are correct :-) ...)